src/platforms/javascript/common/session-replay/index.mdx

```diff
@@ -21,6 +11,8 @@ description: "Learn how to enable Session Replay in your app if it is not alread
 
 <Include name="beta-note-session-replay.mdx" />
 
+Sentry's Replay product is geared toward developers. By default, it'll mask all fields, so no personally identifiable information (PII) is collected, eliminating any concerns with General Data Protection Regulation (GDPR). Learn more on <PlatformLink to="/session-replay/privacy">Session Replay Privacy</PlatformLink>.
+
 With [session replays](/product/session-replay/) you can get to the root cause of an error or latency issue faster by seeing all the technical details related to that issue in one visual replay on your web application. Sentry provides a video-like reproduction of user interactions on a site or web app. All user interactions - including page visits, mouse movements, clicks, and scrolls - are captured, helping developers connect the dots between a known issue and how a user experienced it in the UI.
```

Comment:
Line 16 should really be the first paragraph on this page since this actually explains what the feature is.